### ISSUE TITLE
Calling Java Class Methods from C

### ISSUE THREAD

**Issue by JaroslawWiosna** [JaroslawWiosna/regionalizer#26]
Research task:

In #10 : 
>TODO: make the java files run other way than by Eclipse - this idea actually should be done but don't have any idea now how to do this (maybe some tests and/or travis running them?) - but it is additional functionality only when we will have more time.

Let's analyse if calling java methods in C++ code is possible.

Useful webpages:
[http://hildstrom.com/projects/jni/index.html](http://hildstrom.com/projects/jni/index.html)

**Comment by JaroslawWiosna** [comment]
@kermit10000000 What is the current status?

I think this issue can be moved to DONE when you prepare **wiki** page with articles about it.
(I just wanted to say, that this issue is about *initial investigation*, so even small info would be appreciated, so task/issue can be moved to DONE)

**Comment by kermit10000000** [comment]
I have not started it yet but maybe i will handle it this week.

**Comment by kermit10000000** [comment]
After first investigation it might take more time than I expected due to the need of changes in compilators to run this mix

**Comment by kermit10000000** [comment]
Little update: After looking deeper I have found another way than chaning java files.
I have described on our wiki how to link paths to build the project and some info about how to code in this mix, how to use it, what are differences between C and C++ calling java methods with some code examples.
What I could not do so far was running the code after i have build it. It keeps give me an error that I do not have  .dll file and to reinstall the program(which would not help for sure). I tried a few options so far but it still keeps failing... 
I will keep trying but if I won't find a solution before the next planning I will assume the issue as closed to not waste more time on it. (maybe we will go back to this in the future than when we will need it or some other contributor will know).
